packages/api/src/client/internal/generated/openapi/request/OpenApiRequest.ts

```diff
@@ -0,0 +1,11 @@
+export type OpenApiRequest<TExpectedResponse, TResponse = TExpectedResponse> = (
```

Comment:
Is there an implementation of this somewhere?

Reply:
I can provide an implementation, this was more just to define a contract between the generated functions and what we should pass in